datasets/__init__.py

```diff
@@ -0,0 +1,3 @@
+
+from datasets._version import __version__
+from datasets.utils import Interface
```

Comment:
Not sure about the import name, but it can be changed easily

Reply:
I would stick with longer but more specific, so the user knows what kind of datasets, and so they are able to install alongside other packages. Maybe `pip install pydicom-datasets`, to then import to `pydicom_datasets`?

Reply:
Or seeing pydicom-data below... just call it that! :)

Reply:
Yeah, would really like to avoid the name 'dataset(s)' given the obvious confusion with DICOM datasets.  Maybe `data_store`, `file_data`, `DICOM_file_store`, ... some combination of those kinds of words?

Reply:
`dcmstore`?

Reply:
If the module is called divine store, I would expect it to also be able
to create or otherwise interact with said stores (e.g.  https://cloud.google.com/healthcare/docs/how-tos/dicom). But if it’s just a data library, the name should probably reflect that and be something with data or datasets (branded for pydicom)... unless you again want to advertise that it can be used in other ways external to it.

Reply:
Change to `data_store`